src/parsers/manifest/dash/parse_adaptation_sets.ts

```diff
@@ -334,19 +340,23 @@ export default function parseAdaptationSets(
         isDub = true;
       }
 
-      const isClosedCaption = type === "text" &&
+      const isClosedCaptionPredicate = (accessibility: IScheme) => type === "text" &&
```

Comment:
This also creates a weird identation for the next lines.

Also the `accessibility != null` check should not be needed anymore.